Linux warning

```
legolord208@legolord208-X550LD ~/Coding/Go/src/github.com/getlantern $ go get github.com/getlantern/systray/
# github.com/getlantern/systray
systray/systray_linux.c: In function ‘nativeLoop’:
systray/systray_linux.c:37:2: warning: ‘return’ with no value, in function returning non-void
  return;
  ^
```

Still works, so... nothing big. Just letting you know. Probably change it to `return 0` or something ¯\\\_(ツ)\_/¯

--- a/systray_linux.c
+++ b/systray_linux.c
@@ -35,7 +35,7 @@ int nativeLoop(void) {
 	systray_ready();
 	gtk_main();
 	systray_on_exit();
-	return;
+	return 0;
 }
 
 // runs in main thread, should always return FALSE to prevent gtk to execute it again

